code/DebugBarSilverStripeCollector.php

```diff
@@ -169,8 +169,7 @@ public function getWidgets()
 
         $userIcon = 'user-times';
         $userText = 'Not logged in';
-        if (Member::currentUserID()) {
-            $member = Member::currentUser();
+        if ($member = Member::currentUser()) {
```

Comment:
While I personally don't disagree, isn't PSR-2 to have 1 statement per line?
i.e.
```php
$member = Member::currentUser();
if ($member) {
```
?

Reply:
I don't think there's anything in PSR-2 about that, both examples are valid and I think the differences are up to taste, perhaps.

We can't merge this pull request until https://github.com/maximebf/php-debugbar/pull/360 gets released thought - if you could request changes saying that that'd be nice.